README.md

```diff
@@ -33,12 +67,52 @@ Welcome to the editing environment for the DE-TEL Book.
 First, please change the _config.yml "url" parameter to "http://localhost:4000".
 
 Please do not commit this change!
-Now build the page locally with
+Now build the page locally with the following command:
 
     bundle exec jekyll serve
 
  Navigate your browser to `http://localhost:4000/detel-book/` to check results while you develop.
 
+## How to branch and create a pull request 
+
+Now when you have the writing environment setup locally, you can start creating new or change existing content. We recommend that you work on one piece at a time, as it will be easier to review your contribution. Please, follow this process:
+
+__Step 1.__ In GitBash, navigate to the folder with the local version of the DETEL book repository.
+
+You should  notice that Gitbash displayed the name of your current branch in brachets after the name of the directory. It is (master) by default.
+
+__Step 2.__ Create a new branch
+
+    $ git checkout -b "MyNewBranch"
+
+You should now notice that GitBash has switched to your new branch.
+
+__Step 3.__ Edit!
+
+Now you can create, edit or delete any content of the book. For example, you if you want to create a new chapter, you can create a new file and start editing it. You can use any text editr, like a Notepad app, to edit the file. However, we recommend that you install and use [VS Code](https://code.visualstudio.com/) app. This app is free and has several convenient tools in addition to editing the text itself.
+
+The content of the book is written in Markdown, a simple markup language. To get started with Markdown, check this [starting guide](https://www.markdownguide.org/getting-started/) and this [cheat sheet](https://www.markdownguide.org/cheat-sheet/).
+
+__Step 4.__ Submit your edits for review
+
+After you have made a set of edits, you need to run the following commands in GitBash to submit your changes for review in the online repository:
+
+Add all your changes to the submission:
+
+    $ git add -A
+
+Create a commit:
+
+    $ git commit -m "Short human-readable description of the changes I made should be written here"
+
+Send the commit to the online repository:
+
+    $ git push
+
+__Step 5.__ Wait for review
+
+Now you can wait for one of our administrators to check your contribution and accept or reject it. As soon as it is accepted, the contents of your commit will appear online on the public website. While you are waiting for the review, feel free to work on the next set of edits (e.g., your next chapter).

```

Comment:
We should stress that contributors should start unrelated new work on a new branch and that this new branch must be based on the master branch. This way, they can create individual pull requests for them that we can accept or reject. Each pull request then only contains a single chapter or logical unit.

We should avoid the situation where a contributor just creates one branch and keeps adding content to that branch because we would have to review and accept or reject the entire workload at once, meaning that chapters which are fine would have to wait for changes to other chapters. For the same reasons, contributors should not create a new branch based on their own contributed branch as this would still keep adding to their previous, unreviewed contributions.

Reply:
So, I can update this paragraph, just stressing that the new chapter should be started in a new branch created from the master.

Reply:
Yes, it should suffice to just add a sentence stating this.